### ISSUE TITLE
Lint/CopDirectiveSyntax does not allow disabling with wildcard

### ISSUE THREAD

**Issue by Ben3eeE** [rubocop/rubocop#14044]
## Expected behavior

I expect the following syntax to be allowed and valid:

```ruby
def foo # rubocop:disable Metrics/*
   ...
end
```

## Actual behavior

```bash
W: Lint/CopDirectiveSyntax: Malformed directive comment detected. Cop names must be separated by commas. Comment in the directive must start with --.
        def foo # rubocop:disable Metrics/*
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 files inspected, 1 offense detected
```

## Steps to reproduce the problem

Create this file:
```ruby
# frozen_string_literal: true

a = 1 # rubocop:disable Style/*
```
And run rubocop.

Output is
```bash
:3:7: W: Lint/CopDirectiveSyntax: Malformed directive comment detected. Cop names must be separated by commas. Comment in the directive must start with --.
a = 1 # rubocop:disable Style/*
      ^^^^^^^^^^^^^^^^^^^^^^^^^
:3:7: W: [Correctable] Lint/RedundantCopDisableDirective: Unnecessary disabling of Style department.
a = 1 # rubocop:disable Style/*
```

The Lint/RedundantCopDisableDirective is expected in this case but not the Lint/CopDirectiveSyntax.

## RuboCop version

```
➜  bundle exec rubocop -V
1.75.1 (using Parser 3.3.7.3, Prism 1.4.0, rubocop-ast 1.43.0, analyzing as Ruby 3.4, running on ruby 3.4.1) [arm64-darwin24]
  - rubocop-performance 1.24.0
  - rubocop-rspec 3.5.0
```

**Comment by dvandersluis** [comment]
Use the namespace without a slash or asterisk:

`# rubocop:disable Style`

See https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code for more info.
